SmartStorePerformance/ios/SmartStorePerformance/AppDelegate.m

```diff
@@ -0,0 +1,229 @@
+/*
+ Copyright (c) 2011-present, salesforce.com, inc. All rights reserved.
+ 
+ Redistribution and use of this software in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ * Redistributions of source code must retain the above copyright notice, this list of conditions
+ and the following disclaimer.
+ * Redistributions in binary form must reproduce the above copyright notice, this list of
+ conditions and the following disclaimer in the documentation and/or other materials provided
+ with the distribution.
+ * Neither the name of salesforce.com, inc. nor the names of its contributors may be used to
+ endorse or promote products derived from this software without specific prior written
+ permission of salesforce.com, inc.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR
+ IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
+ WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#import "AppDelegate.h"
+#import "InitialViewController.h"
+#import <React/RCTRootView.h>
+#import <React/RCTBundleURLProvider.h>
+#import <SalesforceAnalytics/SFSDKLogger.h>
+#import <SalesforceSDKCore/SFSDKAppConfig.h>
+#import <SalesforceSDKCore/SFPushNotificationManager.h>
+#import <SalesforceSDKCore/SFDefaultUserManagementViewController.h>
+#import <SalesforceSDKCore/SalesforceSDKManager.h>
+#import <SalesforceSDKCore/SFUserAccountManager.h>
+#import <SalesforceReact/SalesforceReactSDKManager.h>
+#import <SalesforceSDKCore/SFLoginViewController.h>
+
+// Fill these in when creating a new Connected Application on Force.com
+static NSString * const RemoteAccessConsumerKey = @"3MVG9Iu66FKeHhINkB1l7xt7kR8czFcCTUhgoA8Ol2Ltf1eYHOU4SqQRSEitYFDUpqRWcoQ2.dBv_a1Dyu5xa";
+static NSString * const OAuthRedirectURI        = @"testsfdc:///mobilesdk/detect/oauth/done";
+
+@implementation AppDelegate
+
+- (id)init
+{
+    self = [super init];
+    if (self) {
+
+        // Need to use SalesforceReactSDKManager in Salesforce Mobile SDK apps using React Native
+        [SalesforceSDKManager setInstanceClass:[SalesforceReactSDKManager class]];
+
+        [SalesforceSDKManager sharedManager].appConfig.remoteAccessConsumerKey = RemoteAccessConsumerKey;
```

Comment:
Replace these instances of `SalesforceSDKManager` with `SalesforceReactSDKManager`.

Reply:
If I'm not mistaken, all that code comes from the template. Do we need to update it?

Reply:
I'll update it in the templates.

Reply:
Done [here](https://github.com/forcedotcom/SalesforceMobileSDK-Templates/pull/135).